### PR TITLE
Jetty Threadpool

### PR DESCRIPTION
**Pull request by KolombetM**
Just to get an idea

### CODE REVIEW

File: src/foam/nanos/Threadpool.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+foam.CLASS({
```
Comment (jlhughes) [inline src/foam/nanos/Threadpool.js]: license header.

**Review by jlhughes — COMMENTED** [review]
Yep, just like this.

File: src/foam/nanos/Threadpool.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/**
```
Comment (jlhughes) [inline src/foam/nanos/Threadpool.js]: /**
 * @license
 * Copyright 2020 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */

**Review by jlhughes — COMMENTED** [review]
1. Rename Threadpool to ThreadpoolConfig
2. Move Threadpool to jetty package (package in source is correct).
3. Move service nspec to foam/nanos/jetty/services.jrl - and name the nspec (context key) 'jettyThreadpoolConfig' 
4. In HttpServer get ThreadpoolConfig from context. 
Thanks.

File: src/foam/nanos/jetty/ThreadpoolConfig.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+/**
+* @license
+* Copyright 2020 The FOAM Authors. All Rights Reserved.
+* http://www.apache.org/licenses/LICENSE-2.0
+*/
+foam.CLASS({
+  package: 'foam.nanos.jetty',
+  name: 'ThreadpoolConfig',
```
Comment (kgrgreer) [inline src/foam/nanos/jetty/ThreadpoolConfig.js]: Could we rename this JettyThreadPoolConfig so it isn't confused with the NANOS threadpool?

Comment (jlhughes) [reply src/foam/nanos/jetty/ThreadpoolConfig.js]: Yes, that is in my notes above.

Comment (jlhughes) [reply src/foam/nanos/jetty/ThreadpoolConfig.js]: the class itself is already in the jetty package. the renaming was at the service (nspec) level.

**Review by jlhughes — APPROVED** [review]

**Pull request merged**